hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -264,6 +266,17 @@ public AbfsConfiguration getAbfsConfiguration() {
     return this.abfsConfiguration;
   }
 
+  public void access(Path path, FsAction mode)
+      throws AzureBlobFileSystemException {
+    LOG.debug("access for filesystem: {}, path: {}, mode: {}",
+        this.client.getFileSystem(), path, mode);
+    if (!this.abfsConfiguration.isCheckAccessEnabled()) {
```

Comment:
Why is this needed as a configuration? Shouldn't it be always valid?

Reply:
Currently the ABFS access() is a noop. Adding the functionality with a config flag to keep it backward compatible.